tasks/rbd_fio.py

```diff
@@ -49,20 +49,21 @@ def task(ctx, config):
     if config.get('all'):
         client_config = config['all']  
     clients = ctx.cluster.only(teuthology.is_type('client'))
+    rbd_test_dir = teuthology.get_testdir(ctx) + "/rbd_fio_test"
```

Comment:
(and it really ought to be os.path.join(teuthology.get_testdir(ctx), '/rbd_fio_test'))